app/Database/seeds/PermissionSeeder.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
Don't need this, this should automatically be synced https://github.com/nabeelio/phpvms/blob/99f4f3b3d81c900f7977149f2323b680f24034a2/app/Services/Installer/SeederService.php#L99